index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Representations of Verifiable Credentials</h3>
+
+        <p>
+Verifiable Credentials can utilize representations other than the standard one defined in this specification (<code>application/vc+ld+json</code>).
+They can do so provided that a mapping is defined between that representation and the standard one defined in this specifiction,
+so that deployments that choose to do so can map that representation to the standard one and process it accordingly.
+This mapping definition should state whether it is unidirectional or bidirectional.
```

Comment:
```suggestion
Verifiable Credentials can utilize representations other than the standard one defined in this specification (<code>application/vc+ld+json</code>), provided that a mapping is defined between that representation and the standard one. This mapping definition should state whether it is unidirectional or bidirectional.
Deployments that choose to do so, can map the representation to the standard one and process it accordingly.

```

Reply:
> The title of the section and the first sentence are the only things that I find issue with in this PR. The rest of it seems fine.

trying to addresses @msporny 's concern in the first sentence.

Reply:
```suggestion
The standard representation of a Verifiable Credential (<code>application/vc+ld+json</code>) is defined in this specification. Other representations can exist, provided that a mapping is defined between that representation and the standard one. This mapping definition should state whether it is unidirectional or bidirectional.
Deployments can choose to map such other representations to the standard one and process them accordingly.
```